src/main/java/uk/ac/ebi/embl/converter/exception/NonExistingFile.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2025 EMBL - European Bioinformatics Institute
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package uk.ac.ebi.embl.converter.exception;
+
+import java.io.IOException;
+import uk.ac.ebi.embl.converter.cli.CLIExitCode;
+
+public class NonExistingFile extends ExitException {
+    public NonExistingFile(String msg, IOException cause) {
+        super(msg, cause);
+    }
+
```

Comment:
Consider using NoSuchFileException (or) FileNotFoundException and remove this class

Reply:
As described on the previous comment, NonExistingFile extends ExitException, which is required for the error handling system to be able to map this exception to the correct exit code.